docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,218 @@ Postprocessing
+In many cases this comes at the expense of performance, for example, with
+significantly lower accuracy. Regardless, it is a useful data point to compare
+results with.
+
+:class:`ThresholdOptimizer` expects an estimator that provides it with scores.
+While the output of :class:`ThresholdOptimizer` is binary, the input need not
+be. In fact, real valued input, e.g. from a regressor, provides it with many
+more options to create thresholds. For :math:`n` input data points with
+:math:`m \leq n` different score values it has :math:`m+1` different
+thresholds. At each threshold one can create one of two thresholding rules,
+i.e. functions that indicate which data points get label :code:`1` based on
+their score.
+
+We can think of each thresholding rule as a binary classifier and determine
+their true positive rate and false positive rate.
+The code below also visualizes the threshold selection process with an ROC
+curve. The ROC curves consists of the true and false positive rates for each
+of the thresholding rules, with separate ones per sensitive feature value.
+Note that the plot omits points that are within the convex hull of points.
+
+.. plot::
+    :include-source:
+
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from fairlearn.postprocessing import ThresholdOptimizer, plot_threshold_optimizer
+    >>> from sklearn.datasets import fetch_openml
+    >>> import pandas as pd
+    >>> data = fetch_openml(data_id=1590, as_frame=True)
+    >>> X = pd.get_dummies(data.data)
+    >>> y = (data.target == '>50K') * 1
+    >>> sex = data.data['sex']
+    >>> logistic_regression = LogisticRegression()
+    >>> threshold_optimizer = ThresholdOptimizer(estimator=logistic_regression, constraints="equalized_odds", objective="accuracy_score")
+    >>> threshold_optimizer.fit(X, y, sensitive_features=sex)
+    ThresholdOptimizer(constraints='equalized_odds', estimator=LogisticRegression())
+    >>> threshold_optimizer.interpolated_thresholder_._print_thresholds()
+    {
+        "Female": {
+            "p_ignore": 0.13186000824177083,
+            "prediction_constant": 0.035,
+            "p0": 0.9943698649710652,
+            "operation0": "[>0.5]",
+            "p1": 0.005630135028934835,
+            "operation1": "[>-inf]"
+        },
+        "Male": {
+            "p_ignore": 0.0,
+            "prediction_constant": 0.035,
+            "p0": 0.9999261555292187,
+            "operation0": "[>0.5]",
+            "p1": 7.384447078129242e-05,
+            "operation1": "[>-inf]"
+        }
+    }
+    >>> plot_threshold_optimizer(threshold_optimizer)
+
+The printed thresholding rules influence the predictions, so we need to first
+understand the formula that puts all the parts together.
+Instead of selecting a particular thresholding rule we can actually select
+any point between any two thresholding rules.
+This is a very important insight and occurs in almost every case.
```

Comment:
I can't quite figure out what this sentence means. I'm not sure you need it at all.